DuckDuckGo/NavigationBar/View/NavigationBarViewController.swift

```diff
@@ -1011,12 +1011,11 @@ extension NavigationBarViewController: NSMenuDelegate {
             }
             .store(in: &cancellables)
 
-        networkProtectionButtonModel.$showButton
-            .removeDuplicates()
```

Comment:
This is the core fix of the PR. The key change is that we _want_ to allow the publisher to send duplicates, because sometimes other parts of the code may change the value of `isHidden`. So the following situation was possible:

1. The user disables "Show VPN in browser toolbar", sending false here and setting `isHidden` to true
2. User opens the VPN panel which changes `isHidden` to false without affecting the button model
3. User closes the VPN panel which causes the button model to get notified of the popover closing and send false, but `removeDuplicates` was suppressing that since it technically had sent false last time as well